sphinx/developers/r-dev.rst

```diff
@@ -0,0 +1,26 @@
+Using Bio-Formats in R
+======================
+
+OME does not currently provide an R implementation for Bio-Formats.
+However, there are options provided by the community:
+
+Bioconductor RBioFormats
+-------------------------
+
+The `RBioFormats <https://bioconductor.org/packages/devel/bioc/html/RBioFormats.html>`_ 
+project from Bioconductor provides an R package which interfaces the OME Bio-Formats Java library 
+to allow reading of proprietary microscopy image data and metadata.:
+
+.. code-block::
+
+    if (!require("BiocManager", quietly = TRUE))
+      install.packages("BiocManager")
+
+    # The following initializes usage of Bioc devel
+    BiocManager::install(version='devel')
+
+    BiocManager::install("RBioFormats")
+
+
+    # To view documentation for the version of this package installed in your system
+    browseVignettes("RBioFormats")
```

Comment:
Possibly the dev section could include a reference to https://bioconductor.org/packages/devel/bioc/vignettes/RBioFormats/inst/doc/RBioFormats.html which is the API documentation